### PR TITLE
Create GiveMomentum donation page

### PR DESCRIPTION
**Pull request by faihegberg**
Create GiveMomentum donation page as per [card](https://trello.com/c/bi5AgG5F/704-create-a-donation-page-for-givemomentum).

We're not focusing too much on styling in this PR (except for just width and height of the embed). For now we're just focusing on adding an extra page with the embed from GiveMomentum. Code for this new page is mostly the same as our existing Donation page.

### CODE REVIEW

File: src/screens/Donate/Donate.style.tsx [hunk 1]
```diff
@@ -51,6 +51,33 @@ export const EmbedWrapper = styled.div`
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @media (min-width: 600px) {
+    height: 700px;
+  }
+
+  @media (min-width: ${mobileBreakpoint}) {
+    height: 1000px;
```
Comment (chasulin) [inline src/screens/Donate/Donate.style.tsx:71]: id aim to make the iframe non have to be scrollable on the main iphone screen sizes

File: src/screens/Donate/Donate.style.tsx [hunk 2]
```diff
@@ -51,6 +51,33 @@ export const EmbedWrapper = styled.div`
...
+    flex-direction: row-reverse;
+    justify-content: center;
+    padding: 0;
```
Comment (chasulin) [inline src/screens/Donate/Donate.style.tsx:74]: i dont see padding declared anywhere else.. do we need to declare `0` here?

File: src/screens/Donate/Donate.style.tsx [hunk 3]
```diff
@@ -51,6 +51,33 @@ export const EmbedWrapper = styled.div`
...
+ */
+export const GiveMomentumEmbedWrapper = styled.div`
+  height: 700px;
+  width: 100%;
+  max-width: 560px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @media (min-width: 600px) {
```
Comment (chasulin) [inline src/screens/Donate/Donate.style.tsx:66]: `materialSMBreakpoint` is 600px

File: src/screens/Donate/Donate.style.tsx [hunk 4]
```diff
@@ -113,3 +140,7 @@ export const SectionHeader = styled.h2`
   font-weight: bold;
   line-height: 1.4;
 `;
+
+export const StyledIframe = styled.iframe`
+  border: 0;
```
Comment (chasulin) [inline src/screens/Donate/Donate.style.tsx:145]: why nixing the border? the iframe looks pretty floaty/un-contained without a border

File: src/screens/Donate/Donate.style.tsx (hunk 2 above)
Comment (faihegberg) [reply src/screens/Donate/Donate.style.tsx:74]: Nope! This doesn't need to be here. Thanks for pointing it out.

File: src/screens/Donate/Donate.style.tsx (hunk 4 above)
Comment (faihegberg) [reply src/screens/Donate/Donate.style.tsx:145]: No border is just the default style for GiveMomentum. With a border, it current looks like this. I'll play around with it and see if I can style it similar to the GiveButter embed.
image

File: src/App.tsx [hunk 5]
```diff
@@ -35,7 +35,10 @@ const Alerts = lazy(() => import('screens/Learn/Alerts/Alerts'));
 const DataApi = lazy(() => import('screens/DataApi/DataApi'));
 const Terms = lazy(() => import('screens/Terms/Terms'));
 const Privacy = lazy(() => import('screens/Terms/Privacy'));
-const Donate = lazy(() => import('screens/Donate/Donate'));
+const Donate = lazy(() => import('screens/Donate/GiveButterDonate'));
```
Comment (chasulin) [inline src/App.tsx:38]: maybe `GiveButterDonate` instead of `Donate`, since there are now multiple variations?

Comment (faihegberg) [reply src/App.tsx:38]: It is `GiveButterDonate`?

File: src/screens/Donate/Donate.style.tsx [hunk 6]
```diff
@@ -113,3 +128,23 @@ export const SectionHeader = styled.h2`
...
+  }
+
+  @media (min-width: ${mobileBreakpoint}) {
+    height: 750px;
+  }
+
+  @media (min-width: 1200px) {
+    height: 700px;
+  }
+`;
```
Comment (faihegberg) [inline src/screens/Donate/Donate.style.tsx:150]: The height is a bit tricky cos the iframe adjusts its height dynamically. So I tried to get it to adjust with the iframe as much as possible.

File: src/App.tsx (hunk 5 above)
Comment (chasulin) [reply src/App.tsx:38]: `const GiveButterDonate = lazy(() => import('screens/Donate/GiveButterDonate'))`

**Comment by chasulin** [comment]
some widths/alignments [seem to be broken on the give momentum page](https://covid-projections-git-faihegberg-create-give-f6055c-covidactnow.vercel.app/give_momentum_donate)

Screen Shot 2022-07-06 at 6 04 09 PM

**Pull request merged**